breaking-front/src/components/FollowCard/FollowCard.js

```diff
@@ -1,15 +1,24 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import * as Style from 'components/FollowCard/FollowCard.styles';
 import ProfileImage from 'components/ProfileImage/ProfileImage';
 import ImageUrlConverter from 'utils/ImageUrlConverter';
+import { useMutation, useQueryClient } from 'react-query';
+import { deleteUnFollow, postFollow } from 'api/profile';
+import { useTheme } from 'styled-components';
 
-export default function FollowCard({
-  profileData,
-  isPermission,
-  cardClick,
-  deleteClick,
-}) {
+export default function FollowCard({ profileData, isPermission, cardClick }) {
+  const theme = useTheme();
+  const queryClient = useQueryClient();
+  const [isFollow, setIsFollow] = useState(profileData.isFollowing);
+  const { mutate: UnFollow, isLoading: isUnFollowLoading } =
+    useMutation(deleteUnFollow);
```

Comment:
이런 식으로 가면 61번 줄에서 mutate하는 코드가 줄어 가독성이 좋을 것 같습니다. ProfileFollowButton의 useMutation 부분도 이렇게 수정하면 좋겠습니다!
Follow,
```suggestion
  const { mutate: UnFollow, isLoading: isUnFollowLoading } =
    useMutation(deleteUnFollow, {
      onSuccess: () => {
                    queryClient.invalidateQueries('profile');
                    toggleIsFollow();
                  },
      onError: () => {
                    //에러처리
                  },
    });
```